mlops/common/pipeline_utils.py

```diff
@@ -0,0 +1,172 @@
+"""
+This module defines a machine learning pipeline for processing, training, and evaluating data.
+"""
+from azure.identity import DefaultAzureCredential
+from azure.ai.ml.dsl import pipeline
+from azure.ai.ml import MLClient
+import time
+
+from mlops.common.config_utils import MLOpsConfig
+from mlops.common.get_compute import get_compute
+from mlops.common.naming_utils import generate_experiment_name, generate_run_name
+
+
+def set_pipeline_properties(
+    pipeline_job: pipeline,
+    cluster_name: str,
+    display_name: str,
+    build_environment: str,
+    build_reference: str,
+):
+    """
+    """
+
+    pipeline_job.display_name = display_name
+    pipeline_job.tags = {
+        "environment": build_environment,
+        "build_reference": build_reference,
+    }
+
+    # set pipeline level compute
+    pipeline_job.settings.default_compute = cluster_name
+    pipeline_job.settings.force_rerun = True
+    # set pipeline level datastore
+    pipeline_job.settings.default_datastore = "workspaceblobstore"
+
+    return pipeline_job
```

Comment:
Contemplating moving this to the taxi pipelines rather than have it as a utility

Reply:
The point was more to make the prepare and execute functions utilities. These settings are all currently the same which is why I moved them to a common location but it might be the case that a user would want to have these defined differently for each pipeline.. relates back to your other question about moving these to configs

Reply:
Gotcha. I think leaving it per taxi pipeline makes sense then since it can change, but keep the prepare and execute functions pulled out

Reply:
Gonna close this since we got the new issue up about moving the configs which would resolve this